thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/dto/AnomalyFeedbackDTO.java

```diff
@@ -1,8 +1,17 @@
 package com.linkedin.thirdeye.datalayer.dto;
 
+import com.linkedin.thirdeye.anomalydetection.context.AnomalyFeedback;
 import java.io.Serializable;
 import com.linkedin.thirdeye.datalayer.pojo.AnomalyFeedbackBean;
 
-public class AnomalyFeedbackDTO extends AnomalyFeedbackBean implements Serializable {
```

Comment:
Ideally, we don't need to deal with DTO and Bean unless we are creating an anomaly result.
Example:
AnomalyResult anomaly = new MergedAnomalyResultDTO();

After that, we interact with only AnomalyResult.